pandas/core/index.py

```diff
@@ -1473,19 +1490,20 @@ def get_indexer(self, target, method=None, limit=None):
         """
         Compute indexer and mask for new index given the current index. The
         indexer should be then used as an input to ndarray.take to align the
-        current data to the new index. The mask determines whether labels are
-        found or not in the current index
+        current data to the new index.
 
         Parameters
         ----------
         target : Index
-        method : {'pad', 'ffill', 'backfill', 'bfill'}
-            pad / ffill: propagate LAST valid observation forward to next valid
-            backfill / bfill: use NEXT valid observation to fill gap
-
-        Notes
-        -----
-        This is a low-level method and probably should be used at your own risk
+        method : {None, 'pad'/'ffill', 'backfill'/'bfill', 'nearest'}
+            default: exact matches only.
+            pad / ffill: find the PREVIOUS index value if no exact match.
+            backfill / bfill: use NEXT index value if no exact match
+            nearest: use the NEAREST index value if no exact match. Tied
+            distances are broken by preferring the larger index value.
```

Comment:
For the html docs, you have to make bullet points of these (use `-` or `*` before each line) to have this render properly. 
The same for the other docstrings.